src/main/java/org/zone/commands/structure/region/flags/entry/player/ZoneFlagPlayerEntryViewCommand.java

```diff
@@ -49,7 +49,8 @@
         commandContext
                 .getCause()
                 .sendMessage(Identity.nil(),
-                             Component.text("Enabled: " + zone.containsFlag(FlagTypes.PREVENT_PLAYERS)));
+                             Component.text("Enabled: " + zone.containsFlag(FlagTypes.PREVENT_PLAYERS))
+                                     .color(NamedTextColor.AQUA));
```

Comment:
We now have a messages for this message

``commandContext.sendMessage(Messages.getEnabledInfo(zone.containsFlag(FlagTypes.PREVENT_PLAYERS)));``

Reply:
ohoh

Reply:
group?

Reply:
There is one for group too, take a look at the other view commands that have the updated message, but im not too bothered about the group one as we are going to do another sweep of messages